contract.md

```diff
@@ -12,29 +23,77 @@ like to work on, as part of your project (for example, other things related to D
 you have come across, or topics from other classes)? Why are you interested in working on
 these things?_
 
+Calling back to one of the earliest exercises in this class, I'd like to work on
+creating a product not just for me, but for the intended audience. That is, I
+should be continiously checking in with other people to see if what intuitively
+makes sense to me also makes sense to them, and seeing if there are any features
+that I'm missing that they would fine helpful (among other things). I feel like
+when I work on projects, I tend to tunnel-vision on what I want and not others,
+so it would be good to work on that.
+
+This is kind of a stretch goal but I'd also want to work on building a GUI that
+isn't webdev based, just because I've already done a lot of webdev. However,
+this isn't that related to DSLs, hence why I say it's a stretch goal; it might
+be more efficient to just work with what I already know.
+
 ## Time management plan
 
 _How do you plan to set aside time outside of class, to work on the project? Are there
 intermediate milestones that you can create, to help you make consistent progress?_
 
+I'll set dedicated DSL hours so that I can force myself to sit down and work on
+it. I'm thinking one day could be Monday 3pm-5pm. I'll add another day if that
+seems insufficient. 
+
+Here are some milestones that might be helpful:
+- sketch design and what the sheet music might look like. can also sketch gui if
+  helpful
+- create a very barebones gui so that I can start writing the code that will
```

Comment:
You mentioned in class that your priorities for the first week of work are the first two milestones on this list, and I think that's a good plan! It'll be good to assess the GUI stuff early so you can make contingency plans if need-be